### ISSUE TITLE
[Settings> Save]: Shortcut key should be defined for 'Save' and 'Discard Changes' control inside Settings page. 

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#12029]
### Windows Terminal version

1.12.3472.0

### Windows build number

10.0.22504.1010

### Other Software

**Test Environment:**
OS: Windows 11 Version Dev (OS Build 22504.1010)
App: Windows Terminal Preview
Screen Reader: Narrator
Tool: Accessibility Insights for Windows Version 1.1.1741.1

### Steps to reproduce

**Repro Steps:**
1. Open Windows Terminal.
2. Open Settings page using 'Ctr+,'.
3. Make some changes inside given tabs.
4. Verify that whether shortcuts is given to save or discard changes.

**User Experience:**
UX will not be good if Users like Keyboard only, Screen Reader users are not able Save or Discard changes fast.

**Attachments:**
[Shortcut key should be defined for 'Save' and 'Discard Changes' control inside Settings page..zip](https://github.com/microsoft/terminal/files/7773208/Shortcut.key.should.be.defined.for.Save.and.Discard.Changes.control.inside.Settings.page.zip)

### Expected Behavior

Shortcut keys should be defined for 'Save' and 'Discard Changes' control inside Settings page. Shortcut keys should also be included with 'Save' or Discard' button Accessible Name & Tooltip. 
i.e. 'Ctrl+ S' to Save Changes and 'Ctrl + D' to Discard changes.

### Actual Behavior

Shortcut key is not defined for 'Save' and 'Discard Changes' control inside Settings page. Users have to navigate to Save or Discard button in order to save or discard changes.

**Comment by zadjii-msft** [comment]
This one's curious. It would involve specific keybindings that are SUI-specific. That's fine, that makes sense, there are other keybindings that are already TerminalControl-specific, so that's not unreasonable. Binding them by default does get a little dicey. Imagine that someone's got <kbd>ctrl+s</kbd> bound currently to "new tab with Some profile" (as example). If we bound <kbd>ctrl+s</kbd> as `"saveSettingsUI"`, then their binding for `newTab` would override that one. That seems fine, since the explicitly added that setting. It would just be a little confusing that they couldn't have both `saveSettingsUI` and `toggleRetroEffect` (for example) bound to <kbd>ctrl+s</kbd>. Each only works in one context (SUI vs TermControl), but only the last one in the user's settings would get bound. 

> INB4 Dustin and "this is why we need contextual keybindings"

In general, this doesn't seem totally impossible, so let's just go for it.
